fragment.js

```diff
@@ -1,9 +1,13 @@
 "use strict"
 
-function formatSettings() {
-    var settings = ""
+function updateHash() {
+    if (navigationInProgress) {
+        return
+    }
+
+    var settings = "#"
```

Comment:
Don't add the hashmark here. It messes with the "zip" feature. Add it when assigning to `window.location.hash`.